core/src/docs/rfcs/2884_merge_range_read_into_read.md

```diff
@@ -0,0 +1,80 @@
+- Proposal Name: `merge_range_read_into_read`
+- Start Date: 2023-08-20
+- RFC PR: [apache/incubator-opendal#2884](https://github.com/apache/incubator-opendal/pull/2884)
+- Tracking Issue: [apache/incubator-opendal#2811](https://github.com/apache/incubator-opendal/issues/2811)
```

Comment:
＃2811 is the issue for idea. We can create a new one after this RFC has been approved.